Client/Frontend/Brave Notifications/Brave Wallet/WalletNotification.swift

```diff
@@ -33,17 +35,20 @@ class WalletNotification: BraveNotification {
   
   init(
     priority: BraveNotificationPriority,
+    origin: URLOrigin,
     handler: @escaping (Action) -> Void
   ) {
     self.priority = priority
-    self.view = WalletConnectionView()
+    self.origin = origin
+    self.view = WalletConnectionView(origin: origin)
     self.handler = handler
     self.setup()
   }
   
   private func setup() {
     guard let walletPanel = view as? WalletConnectionView else { return }
-    walletPanel.addTarget(self, action: #selector(tappedWalletConnectionView(_:)), for: .touchUpInside)
+    let tapGesture = UITapGestureRecognizer(target: self, action: #selector(tappedWalletConnectionView(_:)))
+    walletPanel.addGestureRecognizer(tapGesture)
```

Comment:
Is this needed because the scroll view messes with the regular `UIControl` tap? Are there any side-effects of using a tap gesture recognizer instead (i.e. from accessibility perspective). Does it still have a button trait?

Reply:
> Is this needed because the scroll view messes with the regular UIControl tap?
Yes

> Are there any side-effects of using a tap gesture recognizer instead (i.e. from accessibility perspective). Does it still have a button trait?

Updated to include the expected accessibility traits now, didn't seem to be working before either.